lib/locations_api_client.py

```diff
@@ -12,10 +13,16 @@ def __init__(self):
         self.logger = create_log('locations_api_client')
         self.url = os.environ['LOCATIONS_API_URL']
 
+        retry_policy = Retry(total=3, backoff_factor=5,
+                             status_forcelist=[500, 502, 503, 504])
+        self.session = requests.Session()
+        self.session.mount('https://', HTTPAdapter(max_retries=retry_policy))
```

Comment:
oo this looks super useful! we have so much homebaked retry in node... i wonder if there are comparable libraries we could use